src/server.c

```diff
@@ -7033,7 +7033,7 @@ int main(int argc, char **argv) {
                 exit(0);
             } else {
                 fprintf(stderr,"Please specify the amount of memory to test in megabytes.\n");
-                fprintf(stderr,"Example: ./redis-server --test-memory 4096\n\n");
+                fprintf(stderr,"Example: ./valkey-server --test-memory 4096\n\n");
```

Comment:
This is the output of a program that we might want to backport to 7.2.x release, without all the other changes. I would prefer this in separat PR as well (or together with changes of the same kind).

Reply:
Sure, updated the diff by removing these line chnages, and now its only variable rename.